src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTextViewRazorDocumentTrackerService.cs

```diff
@@ -16,8 +16,7 @@ namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
     [ContentType(RazorLanguage.ContentType)]
     [TextViewRole(PredefinedTextViewRoles.Editable)]
-    //[Export(typeof(IWpfTextViewConnectionListener))] - we're not exporting this interface because we only want it to be hooked up
-    // in developer mode.
+    [Export(typeof(IWpfTextViewConnectionListener))]
```

Comment:
@NTaylorMullen - I want to enable this so we can start finding issues in it, and build on top of it for other features.

This is of course only going to light up in 15.5